paddle/fluid/framework/tensor_util.h

```diff
@@ -57,8 +57,14 @@ void TensorToVector(const Tensor& src, const platform::DeviceContext& ctx,
 template <typename T>
 void TesnorToVector(const Tensor& src, std::vector<T>* dst);
 
+// copy the result bool to cpu
 bool TensorContainsNAN(const framework::Tensor& tensor);
 bool TensorContainsInf(const framework::Tensor& tensor);
+bool TensorIsfinite(const framework::Tensor& tensor);
+
+void TensorContainsNAN(const framework::Tensor& tensor, framework::Tensor* out);
```

Comment:
Doc the different from previous functions?

Reply:
commented.